daemon/update_linux.go

```diff
@@ -28,5 +28,6 @@ func toContainerdResources(resources container.Resources) libcontainerd.Resource
 	}
 	r.MemoryReservation = uint64(resources.MemoryReservation)
 	r.KernelMemoryLimit = uint64(resources.KernelMemory)
+	r.PidsLimit = uint64(resources.PidsLimit)
```

Comment:
@darkowlzz you need to open a PR against the `v0.2.x` branch of `https://github.com/containerd/containerd` to add that field into `api.proto`. Once that PR is merged you'll need to vendor `containerd` into docker (as part of this PR)